src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
This really doesn't lend to using the tool in uncommon scenarios. For instance, we run the tool as part of Coherence-Signed today (and we used to run it as part of the the partner package mirroring). Relying on tool name as a target seems less magical than relying on the tool putting together the lifecycle.